docs/content/recipes/common/exposing-dataplane-with-metallb.md

```diff
@@ -0,0 +1,119 @@
+## Configure MetalLB for HostedCluster's Data Plane
```

Comment:
can this just be in "how to" as well?

Reply:
It could, but I really think "recipes" section will give this documentation a real value. We can mix things like this, now we are mixing Disconnected + Agent topics, and it's a organisation problem to locate it in one place. Because of that, Toni commented that this recipes section could make much sense.

Another sample for this could be:
- Deploying disconnected operators based on tags instead of digests

This ^^ is a recurring topic I would like to document at some point, and I think the best place is the recipes. WDYT?

Reply:
From my PoV:

- Recipes include things non-related with HCP. (E.G MetalLB to configure services exposure)
- HowTo section implies how to configure HCP to reach a concrete end. (E.G How to configure TLS certificates for a disconnected environment)

In the first bullet point, we include external components to Openshift, I'm saying MetalLB but it could be w/e LoadBalancer you want to configure. 

Another Recipe could be "How to configure Disconnected environments to allow the HostedClsuter to pull images from tag"

Reply:
I'm not sure why this would match the expectation of an external reader. Seems to me quite arbitrary